### PR TITLE
fix: Fixed a bug where horizontal suggested reply items contents are not wrapping to multiple lines.

### PR DESCRIPTION
**Pull request by liamcho**
Fixes: [AC-2521](https://sendbird.atlassian.net/browse/AC-2521)

### Changelogs
- Fixed a bug where horizontal suggested reply items contents are not wrapping to multiple lines.

### After
Screenshot 2024-05-30 at 1 16 39 PM
Screenshot 2024-05-30 at 1 08 47 PM
 
### Figma
https://www.figma.com/design/SVbXU00FhjztekD8AiVukK/UIKit_Work-file_React?node-id=2393-13688&t=cyEn8F21R1sNrdNx-0

[AC-2521]: https://sendbird.atlassian.net/browse/AC-2521?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/GroupChannel/components/SuggestedReplies/index.tsx [hunk 1]
```diff
@@ -11,20 +11,24 @@ export interface SuggestedRepliesProps {
...
-    </div>
+    <>
+      <div
+        className={`sendbird-suggested-replies__option ${type}`}
+        id={value}
+        onClick={(e) => onClickReply(e, value)}
+      >
+        {value}
+      </div>
+    </>
```
Comment (AhyoungRyu) [inline src/modules/GroupChannel/components/SuggestedReplies/index.tsx:31]:
```suggestion
      <div
        className={`sendbird-suggested-replies__option ${type}`}
        id={value}
        onClick={(e) => onClickReply(e, value)}
      >
        {value}
      </div>
```
It seems we don't need this empty tag here

File: src/modules/GroupChannel/components/SuggestedReplies/index.tsx [hunk 2]
```diff
@@ -11,20 +11,24 @@ export interface SuggestedRepliesProps {
...
 }: ReplyItemProps) => {
   return (
-    <div
-      className="sendbird-suggested-replies__option"
-      id={value}
-      onClick={(e) => onClickReply(e, value)}
-    >
-      {value}
-    </div>
+    <>
```
Comment (bang9) [inline src/modules/GroupChannel/components/SuggestedReplies/index.tsx:23]: fragment 없어도 되지 않나요?

File: src/modules/GroupChannel/components/SuggestedReplies/index.tsx (hunk 1 above)
Comment (liamcho) [reply src/modules/GroupChannel/components/SuggestedReplies/index.tsx]: oh weird. I needed it when fixing but now we dont. thanks i removed

**Review by bang9 — APPROVED** [review]

File: src/modules/GroupChannel/components/SuggestedReplies/index.scss [hunk 3]
```diff
@@ -47,14 +47,14 @@
 }
 
 .sendbird-suggested-replies__option {
-  white-space: nowrap;
-  height: 32px;
+  white-space: pre-wrap;
   font-size: 12px;
-  padding: 0 14px;
+  padding: 8px 14px;
```
Comment (AhyoungRyu) [inline src/modules/GroupChannel/components/SuggestedReplies/index.scss:52]: There could be extra padding(or margin) in Widget which used to be covering this. If so, please remove it :)

**Review by AhyoungRyu — APPROVED** [review]

File: src/modules/GroupChannel/components/SuggestedReplies/index.scss (hunk 3 above)
Comment (liamcho) [reply src/modules/GroupChannel/components/SuggestedReplies/index.scss:52]: no widget now uses UIKit SR so no problem

**Pull request merged**